### DISCUSSION TITLE
dfu device how to get serial number

### DISCUSSION THREAD

**Discussion by premanandaembed** [pyusb/pyusb#457] [Q&A]
i'm not getting serial number while device is dfu mode but normal mode i'm getting serial number.

**Comment by mcuee** [comment]
Does the device have serial number in dfu mode?

**Reply by premanandaembed** [reply]
yes

**Reply by mcuee** [reply]
In that case, you may want to post the debug message to see what goes wrong.

**Comment by mcuee** [comment]
Close this old discussion as there is no input from OP.
